web/src/pages/people/index.tsx

```diff
@@ -45,6 +55,30 @@ export default function PeoplePage() {
   const isIAO = userData?.roles.includes("information-asset-owner");
   const isTreOpsStaff = userData?.roles.includes("tre-ops-staff");
 
+  const handleUserSearch = async (query: string) => {
+    if (query !== "") {
+      const response = await getUsers({ query: { find: query } });
+      if (response.response.ok && response.data) {
+        setUsers(response.data);
+      } else {
+        setSearchErrorMessage("Oops, something went wrong with your search. Refresh and try again");
+      }
+    } else {
+      setUsers(originalUsers);
+    }
+  };
```

Comment:
Personal preference: the embedded if statements can be simplified by returning early e.g.

```
if (query === "") {
  setUsers(originalUsers);
  return;
}
...
```

Reply:
https://github.com/ucl-arc-tre/portal/pull/251/commits/045cc2464802aba82c28db257ec209f4a65cd326